tests/persist.rs

```diff
@@ -4,51 +4,42 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-#[cfg(feature = "persistance")]
-mod persist {
+use dusk_hamt::Hamt;
+use microkelvin::Portal;
+use rkyv::rend::LittleEndian;
 
-    use dusk_hamt::Hamt;
-    use microkelvin::{
-        BackendCtor, Compound, DiskBackend, PersistError, Persistance,
-    };
+#[test]
+fn persist_across_threads() {
```

Comment:
Since these are done by `microkelvin` already, shouldn't we ditch the "across_threads" test here?

Reply:
We could, but the persistence in general still needs to be tested, since we need to guarantee that the right bounds work also with persistence.

The across_threads thing is not strictly necessary, but it does not hurt either.

Reply:
Alright. We keep it.